rpcs3/Emu/SysCalls/Modules/cellNetCtl.cpp

```diff
@@ -48,7 +52,15 @@ int cellNetCtlGetState(vm::ptr<u32> state)
 {
 	cellNetCtl.Warning("cellNetCtlGetState(state_addr=0x%x)", state.addr());
 
-	*state = CELL_NET_CTL_STATE_Disconnected; // TODO: Allow other states
+	// TODO: Unix/Linux and Mac implementation
+#ifdef _WIN32
+	if (InternetCheckConnection(L"http://www.google.com", FLAG_ICC_FORCE_CONNECTION, 0))
```

Comment:
This is a really ugly "solution" in my opinion. Also, I don't think whether the host machine is or is not connected to the net should directly effect the emulated system.